smarts/core/renderer.py

```diff
@@ -158,7 +169,16 @@ def __init__(self, simid: str):
         self._vehicle_nodes = {}
         # Note: Each instance of the SMARTS simulation will have its own Renderer,
         # but all Renderer objects share the same ShowBaseInstance.
+
         self._showbase_instance = _ShowBaseInstance()
+        try:
+            print(self._showbase_instance.test_init())
+        except Exception as e:
+            # Known reasons for this failing:
+            raise AttributeError(
+                f"You may not have installed the [render] dependencies required to render the camera sensor observations. "
+                "Install them first using the command `pip install -e .[render]` at the source directory."
+            ) from e
```

Comment:
OK, rather than adding this `test_init()` method it might be better to throw the `RendererException` within the `_ShowBaseInstance.__init()__`. Then we may not need to even need to do this

```suggestion
        self._showbase_instance = _ShowBaseInstance()
```